eth/tracers/internal/tracetest/txnOpCodeTracer_test.go

```diff
@@ -0,0 +1,130 @@
+package tracetest
```

Comment:
Can we move this to `eth/tracers/native` and rename the package to `trace_test`?  Similarly, I would recommend moving the testdata files to `eth/tracers/native/testdata`, as it's more idiomatic.

Reply:
https://blocknative.slack.com/archives/C01H2JUBQ48/p1673554679525629?thread_ts=1673553557.154409&cid=C01H2JUBQ48